build/sources.props

```diff
@@ -7,6 +7,8 @@
       $(RestoreSources);
       https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json;
       https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json;
+      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
```

Comment:
@natemcmaster FYI in case you have an alternative.

Reply:
It's temporary

Reply:
This is fine for local builds and temporary workarounds, but would break in Universe if you don't also update that to pull the new corefxlab packages.